examples/my_examples/refine/unit_test.zls

```diff
@@ -129,6 +129,13 @@ let add103 (x : int{x > 0}) (y : int{y > 0}) : int{(add103_return > 0) && (add10
   x + y 
 
 	(* - no refinement *)
```

Comment:
Make sure to update unit tests and delete the old syntax examples if we are not using them anymore